app/Http/Controllers/AdminDash.php

```diff
@@ -830,6 +830,11 @@ public function storeFile(Request $request) {
     }
 
     public function fileSeparator(Request $request) {
+        $validator = $request->validate([
+            'title' => 'required',
+            'type' => 'required',
+        ]);
+
```

Comment:
Let's add this in saving a file too (line 860). That will prevent a 500 if someone tries saving a file without a title or type.